src/ntapfuse_ops.c

```diff
@@ -183,11 +182,40 @@ ntapfuse_chmod (const char *path, mode_t mode)
 int
 ntapfuse_chown (const char *path, uid_t uid, gid_t gid)
 {
-
     char fpath[PATH_MAX];
+    int res;
+    int usage = 0;
     fullpath (path, fpath);
+    //get file size
+    long fileSize = getFileSize (fpath);
```

Comment:
I am not sure if we already checked the file existence, if not maybe add error handling for file not present.